### ISSUE TITLE
Optimization problem when training SEAM from scratch

### ISSUE THREAD

**Issue by Angusky** [YudeWang/SEAM#12]
Hi, firstly thank you for releasing the code, I've successfully reproduced part of the result by using the provided weights.

However, when I tried to train SEAM from scratch (not using any pretrained weights), it seems ER loss easily goes down to 0 and ECR loss just cannot go down, then the model cannot improve anymore. I've tried to increase the loss weight of ECR loss but the outcome is still the same.
Could you provide more details or suggestions on how you train SEAM without pretrained weights?

Thanks!

**Comment by YudeWang** [comment]
Hi @Angusky ,

Maybe increase the weight of classificaiton loss will be helpful. The classificaiton loss guides the network to preserve the basic shape of CAM, while the ER and ECR loss will revise the basic shape to a better result.

**Comment by Angusky** [comment]
> Hi @Angusky ,
> 
> Maybe increase the weight of classificaiton loss will be helpful. The classificaiton loss guides the network to preserve the basic shape of CAM, while the ER and ECR loss will revise the basic shape to a better result.

Thanks for your quick reply, I will try that. May I ask did you encounter similar optimization problem in your training?

**Comment by YudeWang** [comment]
> Thanks for your quick reply, I will try that. May I ask did you encounter similar optimization problem in your training?

The similar optimization problem may appear in revised CAM if ECR loss is not set correctly.
